src/middleware/errorHandler.js

```diff
@@ -0,0 +1,19 @@
+import Joi from "joi";
+
+const errorHandler = (err, req, res, next) => {
+  const fullUrl = `${req.protocol}://${req.get("host")}${req.originalUrl}`;
+
+  const errorResponse = {
```

Comment:
Please update this by reviewing the structure on our Confluence page.

Reply:
Thanks for the suggestion it is done